.github/workflows/check_verilog.yml

```diff
@@ -0,0 +1,95 @@
+name: check-verilog
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+env:
+  CIRCT_VERSION: 1.72.0
+  CIRCT_ARCHIVE: "circt-full-shared-linux-x64.tar.gz"
+  DEPENDENCIES_DIR: "deps"
+  MLIR_TARGETS_PATH: "lib/cmake/mlir/MLIRTargets.cmake"
+  BUILD_DIR: "build"
+  UTOPIA_EXECUTABLE: "src/umain"
+  MODULE_OUTPUT_PATH: "output.sv"
+  LIBRARY_OUTPUT_PATH: "lib.sv"
+
+jobs:
+  build-and-parse-verilog:
+    strategy:
+      matrix:
+        kernel: [polynomial2, scalar3, matrixmul2, muxmul, addconst, movingsum]
+        config: [1]
```

Comment:
What does it mean? Numbers aren't illustrative, by the way.

Reply:
Fixed with different configuration naming.